src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -83,10 +84,31 @@ public async Task SslStream_SendReceiveOverNetworkStream_Ok()
             listener.Stop();
         }
 
-        [Fact]
+        [ConditionalFact]
```

Comment:
This will always run. But this allows to use SkipTestException in the test for "dynamic" skip.
In general this is hook to our extenstion. If we throw in normal "fact" that would cause test to fail.